recipes/chrony_config.rb

```diff
@@ -16,6 +16,6 @@
 # limitations under the License.
 
 service node['cfncluster']['chrony']['service'] do
-  supports restart: true
+  supports restart: false
```

Comment:
Minor: we could avoid to set it because it is the default. But we can leave it as is to explicitly specify it.